osu.Game/Screens/Multi/RealtimeMultiplayer/RealtimePlayer.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using osu.Framework.Allocation;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Online.RealtimeMultiplayer;
+using osu.Game.Scoring;
+using osu.Game.Screens.Multi.Play;
+using osu.Game.Screens.Ranking;
+
+namespace osu.Game.Screens.Multi.RealtimeMultiplayer
+{
+    public class RealtimePlayer : TimeshiftPlayer
```

Comment:
This inheritance is maybe a bit ehh, but abstracting out further may be too much of muchness... Probably fine to leave be for now until it's a problem.

Reply:
worth a TODO comment at least IMO.